api/src/api/app/apis/simulations_api.py

```diff
@@ -1,38 +1,40 @@
 # coding: utf-8
 
-from typing import Dict, List, Optional  # noqa: F401
+from typing import List, Optional  # noqa: F401
 
+from app.controller.simulation_controller import SimulationController
+from app.models.extra_models import TokenModel  # noqa: F401
+from app.models.gridcell_data import GridcellData
+from app.models.id import ID
+from app.models.movement_data import MovementData
+from app.models.new_scenario import NewScenario
+from app.models.node_migrations_inner import NodeMigrationsInner
+from app.models.scenario import Scenario
+from app.models.simulation_run_status import SimulationRunStatus
+from app.utils.constants import MigrationSort, MovementFilter
 from fastapi import (  # noqa: F401
     APIRouter,
     Body,
     Cookie,
     Depends,
+    File,
     Form,
     Header,
     Path,
     Query,
     Response,
     Security,
-    status,
     UploadFile,
-    File,
+    status,
 )
-
-from app.models.extra_models import TokenModel  # noqa: F401
-from app.models.id import ID
-from app.models.new_scenario import NewScenario
-from app.models.scenario import Scenario
-from app.models.simulation_run_status import SimulationRunStatus
 from security_api import get_token_bearerAuth
 
-from app.controller.simulation_controller import SimulationController
-
 router = APIRouter()
 simulation_controller = SimulationController()
 
 
 @router.post(
-    "/scenarios",
+    "/scenarios/",
```

Comment:
```suggestion
    "/scenarios",
```
Bitte keine "trailing slashes".